_content/operate-connector.md

```diff
@@ -0,0 +1,374 @@
+## Hosting an Interledger node
+
+Interledger users may want to operate their own node software, rather than relying on a third party provider, just like some publishers want to host their own website.
+
+There are three primary implementations to operate an Interledger connector:
+
+- [Java Connector](https://github.com/interledger4j/ilpv4-connector) &mdash; _Actively maintained by Xpring and recommended for new production deployments._
+- [Interledger.rs (Rust)](https://github.com/interledger-rs/interledger-rs) &mdash; _Not actively developed, but recommended to get started for its ease of use and featureset._
+- [`ilp-connector` (JavaScript)](https://github.com/interledgerjs/ilp-connector) &mdash; _Battle-tested in production._
+
+## Overview
+
+In this tutorial, we will spin up a local test network with three Interledger.rs nodes (colloquially, Alice, Bob, and Charlie) and send a cross-currency payment between them, using a local Ethereum testnet and the XRP Ledger testnet to settle with ledger transfers.
```

Comment:
Suggestion: make this into a bulleted list, with each bullet aligning to the detailed steps later in the tutorial. I think this would require restructuring the content a little more--e.g., add a header before Step 1 that's something like "Set up local environment", and so on. But for this specifically, I'm thinking of a change like:

> In this tutorial, we will:
> 
> 1. Spin up a local test network with three Interledger.rs nodes
> 2. Send a cross-currency payment between them
> 3. Settle the payments with ledger transfers

Reply:
Changed this to a list and restructured the tutorial a bit 👍